x-pack/plugins/beats_management/common/domain_types.ts

```diff
@@ -25,9 +25,11 @@ export enum OutputType {
   elasticsearch = 'elasticsearch',
   logstash = 'logstash',
   kafka = 'kafka',
-  console = 'console',
 }
 
+// TODO convery above enum to an array dynamicly here
+export const OutputTypesArray = ['elasticsearch', 'logstash', 'kafka'];
```

Comment:
how about turning this into a predicate? Then you don't need to fix the other code when you work on the todo, please get rid of the lambda:

```
function IsOutputSupported(output:string):bool {
  ...
}
```